linux_os/guide/system/network/network-ipv6/disabling_ipv6/grub2_disable_ipv6/rule.yml

```diff
@@ -0,0 +1,94 @@
+documentation_complete: true
+
+prodtype: rhel7,ol7,rhel8,ol8,fedora,rhv4,ocp4
+
+title: 'Ensure IPv6 is disabled through kernel boot parameter'
+
+description: |-
+    To disable IPv6 protocol support in the Linux kernel,
+    add the argument <tt>ipv6.disable=1</tt> to the default
+    GRUB 2 command line for the Linux operating system in
+{{% if product in ["rhel7", "ol7", "rhv4"] %}}
+    <tt>/etc/default/grub</tt>, so that the line looks similar to
+    <pre>GRUB_CMDLINE_LINUX="... ipv6.disable=1 ..."</pre>
+    In case the <tt>GRUB_DISABLE_RECOVERY</tt> is set to true, then the parameter should be added to the <tt>GRUB_CMDLINE_LINUX_DEFAULT</tt> instead.
+{{% else %}}
+    <tt>/boot/grub2/grubenv</tt>, in the manner below:
+    <pre># grub2-editenv - set "$(grub2-editenv - list | grep kernelopts) ipv6.disable=1"</pre>
+{{% endif %}}
+    
+
+rationale: |-
+    Any unnecessary network stacks - including IPv6 - should be disabled, to reduce
+    the vulnerability to exploitation.
+
+severity: medium
```

Comment:
Why ``medium`` vs ``low``?

Reply:
Nice catch, I think this rule hould have severity: low, as we are only limiting possible attack surface. I will also modify other rules disabling ipv6 in separate PR.